src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,113 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconSize } from "@components/Icon/Icon";
+import { ReactComponent as IconCaretDown } from "@components/Icon/Svg/CaretDown.svg";
+import { ReactComponent as IconCaretUp } from "@components/Icon/Svg/CaretUp.svg";
+import { ReactComponent as IconReject } from "@components/Icon/Svg/Reject.svg";
+import useClickOutside from "@hooks/useClickOutside";
+import { Size } from "@utilities/enum";
+import { ReactElement, useRef, useState } from "react";
+import css from "./Dropdown.module.css";
+import DropdownMenu from "./DropdownMenu/DropdownMenu";
+import DropdownMenuItem, { MenuItem } from "./DropdownMenuItem/DropdownMenuItem";
+
+export interface ItemBlock {
```

Comment:
Move this to `DropdownMenu` otherwise you have a circular dependency between `Dropdown` and `DropdownMenu`. Also I would name it `MenuBlock`.

Reply:
Done 👌🏼